include/crm/pengine/pe_types.h

```diff
@@ -47,6 +47,7 @@ typedef struct resource_object_functions_s {
     char *(*parameter) (pe_resource_t*, pe_node_t*, gboolean, const char*,
                         pe_working_set_t*);
     void (*print) (pe_resource_t*, const char*, long, void*);
+    void (*output) (pe_resource_t*, long, void*);
```

Comment:
This struct is part of the public API, so the new function has to be added to the end of the struct to preserve backward compatibility when using shared libraries

Reply:
Done.